src/helm/ralph/Chart.yaml

```
@@ -22,3 +22,21 @@ version: 0.1.0
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
 appVersion: "3.5.1"
+
+dependencies:
+  ##https://github.com/bitnami/charts/tree/main/bitnami/mongodb
+  - name: mongodb
+    version: 6.0.5
+    repository: oci://registry-1.docker.io/bitnamicharts/mongodb
+    condition: dependencies.mongodb.enabled
+  - name: clickhouse
+    version: 23.4.1
+    repository: https://github.com/bitnami/charts/tree/main/bitnami/clickhouse
```

Why not using the same pattern?
```suggestion
    repository: oci://registry-1.docker.io/bitnamicharts/clickhouse
```